htmresearch/regions/PoolingRegion.py

```diff
@@ -243,7 +249,14 @@ def compute(self, inputs, outputs):
     The guts of the compute are contained in the self._poolerClass compute() call
     """
     activeCells = inputs["activeCells"]
-    predictedActiveCells = inputs["predictedActiveCells"]
+    predictedActiveCells = inputs["predictedActiveCells"] if (
```

Comment:
This is not a required input. In fact in the NLP experiment that uses pooling, it's connected to a sensor directly, which wouldn't produce predicted active cells.